Sources/PointFree/Account/Cancel.swift

```diff
@@ -9,18 +9,25 @@ import Prelude
 import Styleguide
 @testable import Tuple
 
+func requireSubscriptionAndOwner<A>(
```

Comment:
All of the middleware below took several seconds to compile before extracting these shared filters. Now it's under 500 ms.

Reply:
dang. well at least its nice to see how easy it is to define new transformers out of existing!